### PR TITLE
Add MongoDB support

### PR DESCRIPTION
**Pull request by tombates-gds**
Added MongoDB save.  Minor changes to secret handling.

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -152,22 +168,33 @@ router.post('/finish', function(req, res){
...
-      'stage': _userinfo.assessment_stage,
-      'assessment_time': "AM"
+      'service_stage': _userinfo.assessment_stage,
+      'assessment_time': _userinfo.assessment_time,
+      'briefing_deadline': _userinfo.assessment_date
     })
 
-    //notify the team
+    // NOTIFY - SERVICE ASSURANCE TEAM
     notifyClient.sendEmail("22098707-7d16-453a-8e02-baecc466c2d5", team_email, {
```
Comment (rrees) [inline app/routes.js:191]: Is this the API key for the notify service? Should it come from config as well?

File: app/routes.js [hunk 2]
```diff
@@ -2,14 +2,30 @@ var express = require('express')
...
+var dbPassword = process.env.DB_PASSWORD
+var notifyKey = process.env.NOTIFY_KEY
+var team_email = process.env.STANDARDS_TEAM_EMAIL
 
-var notifyClient = new NotifyClient(apiKey)
 
-var team_email = process.env.STANDARDS_TEAM_EMAIL
+// CONNECT TO DB
+var MongoClient = require('mongodb').MongoClient
+var dbURL = 'mongodb://' + dbUser + ':' + dbPassword + '@ds155028.mlab.com:55028/gds-assessments'
```
Comment (rrees) [inline app/routes.js:15]: It is probably worth: a) extracting the URL generation out into a function that takes an environment and returns the URL; b) allowing the host and database name to be configurable as well.

File: app/routes.js [hunk 3]
```diff
@@ -2,14 +2,30 @@ var express = require('express')
...
+var db
```
Comment (rrees) [inline app/routes.js:16]: I think you want to use  `let` here rather than `var`

**Comment by abisola** [comment]
@tombates-gds merged but with need to still take a look at @rrees's recommendations

File: app/routes.js (hunk 1 above)
Comment (tombates-gds) [reply app/routes.js:191]: It's the email template ID so unique to the email we're sending.  Have checked with the Notify team and there's no need to treat as a secret.

**Pull request merged**